exercises/practice/connect/src/test/java/ConnectTest.java

```diff
@@ -0,0 +1,220 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
We have been transitioning tests over to AssertJ - mind switching this one over as well?

The assertion becomes: assertThat(got).isEqualTo(want)